samples/websockets/app.yaml

```diff
@@ -15,6 +15,9 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: tornado
+  labels:
+    app: tornado
+    version: v1
```

Comment:
name and app

Reply:
Removed based on https://github.com/istio/istio/pull/27242#issuecomment-692549416